routers/users.py

```
@@ -0,0 +1,88 @@
+async def delete_me(user_service: Annotated[UserService, Depends(get_auth_service)],
+                    current_user: Annotated[dict, Depends(get_current_user)]):
+    await user_service.delete_user(current_user["email"])
+    return None
+
+@router.patch("/me",response_model=CommonResponse[UserUpdateResponse],status_code=status.HTTP_200_OK)
+async def update_me(user_service: Annotated[UserService, Depends(get_auth_service)],
+                    current_user: Annotated[dict, Depends(get_current_user)],
+                    current_password: str = Form(...),
+                    password: str = Form(),
+                    nickname: str = Form(),
+                    profile_image: UploadFile = File(None)
```

Ruff (0.14.11)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)